openpype/hosts/webpublisher/webserver_service/webpublish_routes.py

```diff
@@ -284,7 +284,7 @@ async def post(self, request) -> Response:
                 args.append("--{}".format(key))
                 # Extend list into arguments (targets can be a list)
                 if isinstance(value, (tuple, list)):
-                    args.extend(value)
+                    args.append(" ".join(value))
```

Comment:
Does this work for `subprocess.Popen` calls? That's not how you should be passing lists of arguments to Popen. Seems odd that this is a fix unless maybe the script it's triggering is not correctly parsing input arguments.

Reply:
I do agree, for what usecase this is?

Reply:
I think the problem is that `click` is expecting each value as new argument ["target1", "target2"] should be `-t target1 -t target2` instead of `-t target1 target2`.

This should work:
```python
        for key, value in add_args.items():
            # Skip key values where value is None
            if value is None:
                continue
            arg_key = "--{}".format(key)
            if not isinstance(value, (tuple, list)):
                args.extend([arg_key, value])
                continue
                
            for item in value:
                args.extend([arg_key, item])
```